lib/twap/index.js

```diff
@@ -51,6 +51,7 @@ const config = require('./config')
 const TWAP = defineAlgoOrder({
   id: 'bfx-twap',
   name: 'TWAP',
+  headersForLogFile: ['_', 'PRICE', 'TOTAL'],
```

Comment:
we can add ID here, this way people are not puzzling what `_` means.

ID, PRICE, TOTAL